os/net/netmgr/bsd_socket_api.c

```diff
@@ -23,21 +23,25 @@
  */
 
 #include <tinyara/config.h>
-#include <tinyara/cancelpt.h>
 
 #ifdef CONFIG_NET
-
+#include <tinyara/cancelpt.h>
 #include <sys/socket.h>
 #include <sys/types.h>
 #include <netinet/in.h>
 #include <net/if.h>
 #include <tinyara/lwnl/lwnl.h>
 #include "netstack.h"
+#include "uds.h"
 
-/*  Private */
+#define CONFIG_NUDS_DESCRIPTORS 0 // to be fixed: temporary
```

Comment:
```suggestion
#ifndef CONFIG_NUDS_DESCRIPTORS
#define CONFIG_NUDS_DESCRIPTORS 0 // to be fixed: temporary
#endif
```
Would you change it tmporarily?
I'll remove this line after adding CONFIG_NUDS_DESCRIPTORS in Kconfig.